src/NewsService.Mappers/Models/NewsInfoMapper.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.Models.Broker.Requests.Company;
+using LT.DigitalOffice.Models.Broker.Requests.User;
+using LT.DigitalOffice.Models.Broker.Responses.Company;
+using LT.DigitalOffice.Models.Broker.Responses.User;
+using LT.DigitalOffice.NewsService.Mappers.Models.Interfaces;
+using LT.DigitalOffice.NewsService.Models.Db;
+using LT.DigitalOffice.NewsService.Models.Dto.Models;
+using MassTransit;
+using Microsoft.Extensions.Logging;
+
+namespace LT.DigitalOffice.NewsService.Mappers.Models
+{
+  public class NewsInfoMapper : INewsInfoMapper
+  {
+    private IRequestClient<IGetUserDataRequest> _userRequestClient;
+    private IRequestClient<IGetDepartmentRequest> _departmentRequestClient;
+    private readonly ILogger _logger;
+
+    private string GetUserFullName(Guid userId)
+    {
+      string fullName = null;
+
+      try
+      {
+        var request = IGetUserDataRequest.CreateObj(userId);
+        var response = _userRequestClient.GetResponse<IOperationResult<IGetUserDataResponse>>(request).Result;
+
+        if (!response.Message.IsSuccess)
+        {
+          _logger.LogWarning($"Can't found user FIO. Reason: '{string.Join(',', response.Message.Errors)}'");
```

Comment:
you cannot find user, not his fio and i don't think so that English contains abbreviation FIO

Reply:
done